Personal_Portofolio.html

```diff
@@ -67,7 +64,7 @@ <h2 class="projects-section-header">These are some of my projects</h2>
     <section id="contact" class="contact-section">
       <div class="contact-section-header">
         <h2 >Let's work together...</h2>
-        <p style="font-size: 19px; color: white"><i>How do you take your coffee?</i></p>
+        <p style="font-size: 19px; color: blue"><i>How do you take your coffee?</i></p>
```

Comment:
🔳 Please consider removing any inline styling to the element as it is mandatory to stick to the stylesheet when styling elements.